diff --git a/node_modules/react-native-mmkv/android/CMakeLists.txt b/node_modules/react-native-mmkv/android/CMakeLists.txt
index 8784d6b..63590e8 100644
--- a/node_modules/react-native-mmkv/android/CMakeLists.txt
+++ b/node_modules/react-native-mmkv/android/CMakeLists.txt
@@ -14,6 +14,7 @@ include_directories(
 
 add_library(reactnativemmkv  # <-- Library name
         SHARED
+        "${NODE_MODULES_DIR}/react-native/ReactCommon/jsi/jsi/jsi.cpp"
         src/main/cpp/cpp-adapter.cpp
         src/main/cpp/MmkvHostObject.cpp
 )
@@ -32,13 +33,13 @@ find_library(
         log
 )
 find_library(
-        JSI_LIB
+        # JSI_LIB
         jsi
         PATHS ${LIBRN_DIR}
         NO_CMAKE_FIND_ROOT_PATH
 )
 find_library(
-        REACT_NATIVE_JNI_LIB
+        # REACT_NATIVE_JNI_LIB
         reactnativejni
         PATHS ${LIBRN_DIR}
         NO_CMAKE_FIND_ROOT_PATH
@@ -48,7 +49,7 @@ target_link_libraries(
         reactnativemmkv
         core
         ${log-lib}
-        ${JSI_LIB}
-        ${REACT_NATIVE_JNI_LIB}
+        # ${JSI_LIB}
+        # ${REACT_NATIVE_JNI_LIB}
         android
 )
